pkg/log/context_test.go

```diff
@@ -64,6 +63,7 @@ var _ = Describe("log", func() {
 				expectPanic(&Settings{
 					Level:  "invalid",
 					Format: "text",
+					Output: os.Stderr.Name(),
```

Comment:
When log level is not recognised, we better fallback to working (safe) log level (like "error") instead crash.

Reply:
we wont crash - it will return 500 - but returning 400 will be better